tests/lib/fakeinstaller/main.go

```diff
@@ -145,7 +187,7 @@ func postSystemsInstallFinish(cli *client.Client, details *client.SystemDetails,
 
 // createAndMountFilesystems creates and mounts filesystems. It returns
 // an slice with the paths where the filesystems have been mounted to.
-func createAndMountFilesystems(bootDevice string, volumes map[string]*gadget.Volume) ([]string, error) {
+func createAndMountFilesystems(bootDevice string, volumes map[string]*gadget.Volume, encryptionChange *client.Change) ([]string, error) {
```

Comment:
once we have another layer it would seem to make more sense to pass that in, instead of the whole change